config/config.go

```diff
@@ -371,10 +372,22 @@ func GetGlobalConfig() *Config {
 
 // Load loads config options from a toml file.
 func (c *Config) Load(confFile string) error {
-	_, err := toml.DecodeFile(confFile, c)
+	metaData, err := toml.DecodeFile(confFile, c)
 	if c.TokenLimit <= 0 {
 		c.TokenLimit = 1000
 	}
+
+	// If any items in confFile file are not mapped into the Config struct, issue
+	// an error and stop the server from starting.
+	undecoded := metaData.Undecoded()
+	if len(undecoded) > 0 && err == nil {
+		undecodedItems := ""
+		for _, item := range undecoded {
+			undecodedItems += fmt.Sprintf("  %s \n", item.String())
```

Comment:
how about:
```suggestion
			undecodedItems += "\t"+%s+"\n"
```
It would be more efficient than `fmt.Sprintf()`

Reply:
I find printf much easier to read than the concatenation of lots of strings, and this is only executed when issuing an error that is going to cause the server to exit so I think the efficiency argument is not very compelling, but it's not a big deal to me either way! However concatenating %s there isn't going to do any good, so I can't commit your suggestion as it is.